testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -67,7 +67,7 @@ public final class Standard
 {
     private static final Logger log = Logger.get(Standard.class);
     private static final int COLLECTOR_UI_PORT = 16686;
-    private static final int COLLECTOR_GRPC_PORT = 4317;
+    private static final int COLLECTOR_HTTP_PORT = 4318;
```

Comment:
do those changes have production implications in default setup?

Reply:
No. It doesn't

Reply:
Is this change necessary for airlift upgrade or just a change we're making because new airlift makes it possible ?

Reply:
The latter